### PR TITLE
Dammit install and execute instructions

### PR DESCRIPTION
**Pull request by ctb**

### CODE REVIEW

**Comment by johnsolk** [comment]
Do we want to use @camillescott's new refactor/1.0 version?

**Comment by ctb** [comment]
Sure - any details?

--
Titus Brown, ctbrown@ucdavis.edu

**Comment by johnsolk** [comment]
https://github.com/camillescott/dammit/tree/refactor/1.0

Uses BUSCO v2, install already in #25

WIll add more install plus usage details tomorrow.

**Comment by ctb** [comment]
I'm not sure I understand --

is the latest dammit release 1.0? or is it not yet released?

now I see that the title for

www.camillescott.org/dammit/

is for version 0.3.1?

where's the canonical dammit release info anyway, then?

**Comment by ctb** [comment]
bump?

**Comment by johnsolk** [comment]
Ok, looks like @camillescott has not officially released 1.0. So, should probably use 0.3.2 for now then, right?

https://github.com/camillescott/dammit/releases

**Comment by johnsolk** [comment]
Not sure if this is the right way to submit changes to existing PR, but just updated `5-annotating.rst` with refactor/1.0 install and command.

**Comment by johnsolk** [comment]
ping @camillescott ?

**Comment by ctb** [comment]
Exactly right, @ljcohen!

I'll take a look at it later today.

File: 5-annotating.rst [hunk 1]
```diff
@@ -1,4 +1,155 @@
...
+    
+Download a custom *Nematostella vectensis* protein database available from JGI:
+
+::
+
+    curl -LO ftp://ftp.ebi.ac.uk/pub/databases/reference_proteomes/QfO/Eukaryota/UP000001593_45351.fasta.gz
+    #curl -LO ftp://ftp.jgi-psf.org/pub/JGI_data/Nematostella_vectensis/v1.0/annotation/proteins.Nemve1FilteredModels1.fasta.gz
+    gunzip proteins.Nemve1FilteredModels1.fasta.gz
+
+Putnam NH, Srivastava M, Hellsten U, Dirks B, Chapman J, Salamov A, Terry A, Shapiro H, Lindquist E, Kapitonov VV, Jurka J, Genikhovich G, Grigoriev IV, Lucas SM, Steele RE, Finnerty JR, Technau U, Martindale MQ, Rokhsar DS. (2007) Sea anemone genome reveals ancestral eumetazoan gener epertoire and genomic organization. Science. 317, 86-94.
```
Comment (standage) [inline 5-annotating.rst]: `s/gener epertoire/gene repertoire/`

File: 5-annotating.rst [hunk 2]
```diff
@@ -1,4 +1,155 @@
...
+
+If a protein dataset is available, this can also be supplied to the ``dammit`` pipeline with ``--user-databases`` as optional evidence for annotation. 
+
+In addition, `BUSCO <http://busco.ezlab.org/>`__ v2 is run (just like in ``4-evaluating-assembly.rst`` -- NOTE maybe this is not necessary to run in step 4?), which will compare the gene content in your transcriptome with a lineage-specific data set. The output is a proportion of your transcriptome that matches with the data set, which can be used as an estimate of the completeness of your transcriptome based on evolutionary expectation (`Simho et al. 2015 <http://bioinformatics.oxfordjournals.org/content/31/19/3210.full>`__). There are several lineage-specific datasets available from the authors of BUSCO. We will use the ``metazoa`` dataset for this transcriptome.
+
+Install stuff
+=============
```
Comment (standage) [inline 5-annotating.rst:14]: We'll want to split this out into installation vs execution, but shouldn't block merging IMO.

**Review by standage — COMMENTED** [review]
LGTM. There's a minor typo that I'll take care of and merge.

**Review by standage — APPROVED** [review]

**Pull request merged**